components/ruby/lib/chef-licensing/license_key_fetcher/chef_licensing_interactions.yaml

```diff
@@ -301,13 +301,26 @@ interactions:
       "3. Commercial License\n": commercial_license_selection
       "4. Quit license generation": exit
 
-  # TODO: Fix the message and update link with the actual license generation link
+  # Note: The below link is specific to InSpec; as per PO there will be different link for other products.
+  #       We need to update the link for other products once we have the link or update to a common link.
   free_trial_license_selection:
-    messages: ["Complete the form available at <%= input[:pastel].blue.underline.blue(\"https://www.chef.io\") %>\n",
-      [Quit]
+    messages: ["<%= input[:pastel].yellow(\"!\") %> Kindly complete the user registration at <%= input[:pastel].blue.underline.blue(\"https://www.chef.io/licensing/inspec/license-genertation-free-trial\") %>\nOnce you submit the details, you will receive the license ID on the email id you provided.\n\nSelect an option",
+      [
+      Validate license now,
+      Quit and validate license later
+      ]
     ]
     prompt_type: "select"
-    paths: [exit_with_message]
+    paths: [validate_license_later_message, ask_for_license_id]
+    response_path_map:
+      "Validate license now": ask_for_license_id
+      "Quit and validate license later": validate_license_later_message
+
+  validate_license_later_message:
+    messages: |
+      You can validate the license later on by selecting <%= input[:pastel].bold("'I already have a license ID'") %> when prompted for license.
+      License can also be validated by running <%= input[:pastel].bold("#{ChefLicensing::Config.chef_executable_name} license add") %> command and then selecting <%= input[:pastel].bold("'Validate a generated license ID option'") %>.
```

Comment:
I know this has been discussed with UX and Lokesh. But I am just curious, why are we only recommending adding a license using `inspec license add` - an interactive way of adding a license? 

Instead of mentioning this, we could have redirected the user to some doc that explains multiple ways of adding a license. Including ENV and argument?
Lot of customer would want to use an automated way of adding licenses in system.